src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -383,6 +394,10 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
         --paper-progress-active-color: var(--accent-color);
         --paper-progress-container-color: rgba(200, 200, 200, 0.5);
       }
+
+      .progress:hover {
+        --paper-progress-height: 8px;
```

Comment:
I am assuming this isn't the best way to do this. I wanted to make sure the user knows that they are on the progress bar. So updating the height would show them clicking will perform an action.

I tried using a calculation of the var height * 2 but that didn't seem to work

Reply:
I think it is fine like this, you could add a new var if you really wanted...